docs/book/v3/naming-strategy/composite-naming-strategy.md

```diff
@@ -3,6 +3,7 @@
 `Laminas\Hydrator\NamingStrategy\CompositeNamingStrategy` allows you to specify which naming
 strategy should be used for each key encountered during hydration or extraction.
 
+<!-- markdownlint-disable-next-line single-h1 single-title -->
```

Comment:
Not required here, as the associated heading is in the wrong order.

Reply:
@froschdesign https://docs.laminas.dev/laminas-hydrator/v3/naming-strategy/composite-naming-strategy/ I see this doc has both CompositeNamingStrategy and Basic Usage as H1. Still should I make Basic Usage as H2?

Reply:
It must be `h2`.

Reply:
Updated